redistools/conn.py

```diff
@@ -2,6 +2,12 @@
 from redis import StrictRedis
 from redis.sentinel import Sentinel
 
+TRACE = os.getenv('TRACE_REDIS', 'false') == "true"
```

Comment:
prob want a `.lower()` here just in case, or do what Alex does an do `"true" in os.getenv..`

Reply:
added the `lower()` call!